SpaDES4Dummies.Rmd

```diff
@@ -180,9 +200,11 @@ abundanceInit <- function(sim) {
 }
 ```
 
+Since we are only interested in simulating and plotting species abundances, we can remove unnecessary events from the script. Only 3 events were kept: the initialisation, an abundance simulation event and a plotting event.
 
+Let's look at the **event functions** (`doEvent.moduleName`, in our dummy example, `doEvent.speciesAbundance` function) first. The event functions were renamed from the template as to reflect my module's actions. Therefore, `Init` became `abundanceInit`, `Plot` became `abundancePlot` and `event1` became `abundanceSim`.
```

Comment:
I think the name changes should either come before the initialisation, or separately in each event.